bin/sandbox

```diff
@@ -49,18 +49,12 @@ fi
 
 cd ./sandbox
 cat <<RUBY >> Gemfile
-gem 'solidus', github: 'solidusio/solidus', branch: '$BRANCH'
-gem 'solidus_auth_devise', '>= 2.1.0'
-gem 'rails-i18n'
-gem 'solidus_i18n'
-
```

Comment:
I think this error has been introduced by the script that tries to copy the extensions' template in order to update the extensions. If we add a comment here that explain that the list of individual gems has a specific purpose, it will less likely happen again.

Reply:
Good point 👍

Reply:
Force-pushed an update ✅